lib/v2/screens/explore_screens/plant_seeds/interactor/viewmodels/plant_seeds_state.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+
+/// --- STATE
+class PlantSeedsState extends Equatable {
```

Comment:
are you not saving the seeds amount the user is entering?

Reply:
Not yet I'll do in next PR

Reply:
kk